csxtools/utils.py

```diff
@@ -1,9 +1,6 @@
 import numpy as np
 import time as ttime
-try:
-    from databroker import get_images
-except ImportError:
-    from dataportal import get_images
+from databroker import get_images
```

Comment:
That does not appear to have **any** of our software stack installed. What does `!which conda` report?